globals.go

```diff
@@ -226,6 +226,7 @@ func IsFailedTaskStatus(status string) bool {
 // evergreen package names
 const (
 	UIPackage      = "EVERGREEN_UI"
+	GQLPackage     = "EVERGREEN_GQL"
```

Comment:
There's no need to add this really, I would just use UIPackage. It's to differentiate between things that are user vs. system-initiated